ros-modified-packages/README.md

```diff
@@ -0,0 +1,26 @@
+# ROS Modified Packages
```

Comment:
Can I add `pre-commit` to this repository as well?

Reply:
Ah, did you mean `pre-commit` template? I meant to run `pre-commit` for the files in this repository to unify the format.

Reply:
https://github.com/tier4/github-actions/pull/3

Reply:
Could you apply `pre-commit` and resolve conflicts? :pray:

Reply:
Now I get it. I approved that PR.

Reply:
Fixed it. https://github.com/tier4/github-actions/pull/2/commits/86eb55158a537c0e34d821e3c4c3755cd507eaf1